pulpcore/pulpcore/app/models/content.py

```diff
@@ -91,6 +91,17 @@ def save(self, *args, **kwargs):
             self.file.close()
             raise
 
+    def delete(self, *args, **kwargs):
+        """
+        Deletes Artifact model and the file associated with the Artifact
+
+        Args:
+            args (list): list of positional arguments for Model.delete()
+            kwargs (dict): dictionary of keyword arguments to pass to Model.delete()
+        """
+        super().delete(*args, **kwargs)
+        self.file.delete(save=False)
```

Comment:
Why ```save=False```?  Because the model has already been deleted?

Do you think maybe these lines should go in opposite order?

Reply:
With `save=False`, the model was being saved as a new record in the database. I'm assuming that we want to delete the file after we delete the model in case the model delete fails which would leave us with a record that has no file.

Reply:
That makes sense.  Approved!